internal/config/properties.go

```diff
@@ -80,48 +84,72 @@ func LoadProperties(env string, cm ...*v1.ConfigMap) error {
 		return fmt.Errorf("config map cannot be nil")
 	}
 
-	var awsAccountID string
-	var err error
+	allowedPolicyAction := strings.Split(cm[0].Data[propertyIamPolicyWhitelist], separator)
+	restrictedPolicyResources := strings.Split(cm[0].Data[propertyIamPolicyBlacklist], separator)
+	restrictedS3Resources := strings.Split(cm[0].Data[propertyIamPolicyS3Restricted], separator)
+	awsMasterRole := cm[0].Data[propertyAwsMasterRole]
+
+	Props = &Properties{
+		allowedPolicyAction:       allowedPolicyAction,
+		restrictedPolicyResources: restrictedPolicyResources,
+		restrictedS3Resources:     restrictedS3Resources,
+		awsMasterRole:             awsMasterRole,
+	}
+
+	//Defaults
+	isWebhook := cm[0].Data[propertWebhookEnabled]
+	if isWebhook != "" {
+		Props.isWebhookEnabled = isWebhook
```

Comment:
Why don't we change logic to 
```
if isWebhook == "true" {
     Props.isWebhookEnabled = true
} else {
     Props.isWebhookEnabled = false
}
```
where isWebhookEnabled is a boolean 

This way we cover both cases of 
1. No value for webhook.enable is set in config-map then default is false
2. By mistake user entering junk value like `yes`, `xyz`

Also, now we can simplify below function
```
func (p *Properties) IsWebHookEnabled() bool {
	resp := false
	if p.isWebhookEnabled == "true" {
		resp = true
	}
	return resp
}
```
to 
```
func (p *Properties) IsWebHookEnabled() bool {
	return p.isWebHookEnabled
}
```

Reply:
Fair. i will change it